### PR TITLE
Add accessibility to the project

### PR DESCRIPTION
**Pull request by Kajal19-del**
# In this project I did the following activities:
- I create the repository with the name "portfolio-project"
- I add accessibility to the project
- I add the webpage title ( portfolio project)
    https://github.com/Kajal19-del/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L7-R7

- I improve alternative text (alt) for images
https://github.com/Kajal19-del/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L89-R90

    https://github.com/Kajal19-del/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L116-R117

- Text headings - no issues found
- I add labels inside the form
    https://github.com/Kajal19-del/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L116-R117
- Color contrast - no issues found (used the same as Figma template)
- Resize - no issues found
- Moving content - no issues found

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Kajal19-del , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid

You have submitted a project, that has no linters set up, Kindly set the linters config files and install the required dependencies and from there you may request another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by dialite** [comment]
Just resolved the pulled request mix-up with my partner. This is the initial pull request.

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio Project</title>
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly change your page title to something that describe the content of the page very well. Something like `Kajal Portfolio` or `Kajal full-stack webdeveloper` is more appropriate than `Portfolio Project`

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Kajal19-del and @dialite, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Accessibility check performed. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark:



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] While trying to test for the button contrast check of your page, I noticed that your logo `Kajal` is failing for default, hover, and focus effects. Kindly check them and make them pass. You can use this [button contrast checker](https://www.aditus.io/button-contrast-checker/) to test for the button contrast of your page

fail-1


- [ ] The test for the `See Project` button contrast check also failed on hover effects. Kindly check it and make it pass. You can use this [button contrast checker](https://www.aditus.io/button-contrast-checker/) to test for the button contrast of your page

fail-2

- [ ] The test for the `Get In Touch` button contrast check also failed on default, hover, and focus effects. Kindly check them and make them pass. You can use this [button contrast checker](https://www.aditus.io/button-contrast-checker/) to test for the button contrast of your page

fail-3

File: index.html (hunk 1 above)
Comment (Kajal19-del) [reply index.html]: @Qoosim I think the review is not valid. All of the code reviewers ignore these color contrast you told me to change. Everybody using the figma template colors, which also fails the color contrast test.
here is an example also, please you can check...
https://github.com/Willian-Berrocal/Portfolio-project/pull/9

I will appreciate it if better light is shown on how to implement the color contrast correction. Thanks.

Comment (Qoosim) [reply index.html]: Hi @Kajal19-del, color contrast check is one of the requirements we test for while reviewing this milestone of your project. The fact that some code reviewers ignore this does not render this review invalid as you've qualified it. I have reviewed a lot of projects on this milestone which the owners manage to make their color contrast pass. You can request for another review. If the next reviewer ignore it or not, all the best. But running the color contrast check on your page is what we're asked to do as part of review. Thanks
You can as well check this [PR](https://github.com/sebalp100/Portfolio/pull/8) to know how students make their color contrast pass.

**Review by frank1738 — APPROVED** [review]
Hi @Kajal19-del , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

- [OPTIONAL ] I would suggest that you use this format `[name of the link](actual link address)` to write your links on the Readme file and PR Descriptions to have a neat documentation

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**